plugins/auth/fps_auth/models.py

```diff
@@ -1,15 +1,11 @@
 import uuid
-from typing import Dict, List, Optional
+from typing import Optional
 
 from fastapi_users import schemas
-from pydantic import BaseModel
+from fps_auth_base.models import BaseUser  # type: ignore
```

Comment:
If you add a [py.typed](https://github.com/jupyter-server/jupyverse/blob/b4b9e5f6663984592d3db6d54a30960def304ef7/plugins/contents/fps_contents/py.typed) in `fps_auth_base`, we could get rid of this `# type: ignore`.